prod.js

```diff
@@ -0,0 +1,3 @@
+var config = require('./config/config.js');
```

Comment:
Use 'node prod.js'. The whole production deployment flow requires some thinking still, as it should not depend on any dev stuff. Now something like 'git checkout && npm install && gulp build && node prod.js' works.

-----Original Message-----
From: "Pekko Lipsanen" <notifications@github.com>
Sent: ‎12/‎10/‎2015 12:39
To: "teekkarispeksi/nappikauppa2" <nappikauppa2@noreply.github.com>
Cc: "Arttu Voutilainen" <arttu.voutilainen@gmail.com>
Subject: Re: [nappikauppa2] Changes required to make reverse proxy work (#27)

In prod.js:
> @@ -0,0 +1,3 @@
> +var config = require('./config/config.js');
Where is this file referenced? How is this used?
—
Reply to this email directly or view it on GitHub.